### PR TITLE
adds uploading of additional documents when kyc request is pending

### PR DESCRIPTION
**Pull request by huesforalice**

### CODE REVIEW

File: app/components/kyc/business/BeneficialOwner.tsx [hunk 1]
```diff
@@ -123,7 +123,7 @@ export class KYCBeneficialOwnerComponent extends React.Component<IProps> {
       <AccordionElement title={name} isOpened={true}>
         <KYCEnhancedForm {...this.props} />
         <MultiFileUpload
-          layout={"personal"}
+          layout={"individual"}
```
[the remark this answers is not in the document]
Comment (huesforalice) [reply app/components/kyc/business/BeneficialOwner.tsx]: Fixed. Do a project search with {" though, you will be surprised :)

File: app/components/kyc/shared/AddDocuments.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+
+import { compose } from "redux";
```
[the remark this answers is not in the document]
Comment (huesforalice) [reply app/components/kyc/shared/AddDocuments.tsx]: I think I have done that now, let me know if it looks right :)

File: app/components/kyc/Kyc.tsx [hunk 3]
```diff
@@ -5,19 +5,25 @@ import { Col, Row } from "reactstrap";
...
 import { KycPanel } from "./KycPanel";
+import { KYCAddDocuments } from "./shared/AddDocuments";
 
 import * as arrowLeft from "../../assets/img/inline_icons/arrow_left.svg";
 
 interface IStateProps {
   requestLoading?: boolean;
   requestStatus?: TRequestStatus;
   redirectUrl: string;
+  pendingRequestType: "individual" | "business" | undefined;
```
[the remark this answers is not in the document]
Comment (huesforalice) [reply app/components/kyc/Kyc.tsx]: now done

File: app/components/kyc/shared/AddDocuments.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  filesLoading: boolean;
+  files: IKycFileInfo[];
+}
+
+interface IDispatchProps {
+  onDropFile: (file: File) => void;
+}
+
+interface IOwnProps {
+  uploadType: "business" | "individual";
```
[the remark this answers is not in the document]
Comment (huesforalice) [reply app/components/kyc/shared/AddDocuments.tsx]: now done

File: app/components/kyc/shared/AddDocuments.tsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+interface IProps {
+  layout: TUploadListLayout;
+}
+
+export const KYCAddDocumentsComponent: React.SFC<
+  IProps & IStateProps & IDispatchProps & IOwnProps
+> = props => (
```
[the remark this answers is not in the document]
Comment (huesforalice) [reply app/components/kyc/shared/AddDocuments.tsx]: done

File: app/components/kyc/business/BeneficialOwner.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (krzkaczor) [reply app/components/kyc/business/BeneficialOwner.tsx]: I KNOW :(

**Pull request merged**